From 685a079201a786a4cee5b054d3ef36ac1dfe5d68 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 21 May 2019 18:09:59 +0300
Subject: [PATCH] [SW-0] preboot: pcie: add "waiting for link" msg

When preboot (actually) waits for the PCIe link, there is no
appropriate message printed and it might seem that it is just hanged.
Add a message to indicate the preboot boot status.

Change-Id: I99cff349ff8735d3bb89e49058a34d032b886ef9
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/common/preboot.c b/common/preboot.c
index 680c56c..bfe7083 100644
--- a/common/preboot.c
+++ b/common/preboot.c
@@ -95,6 +95,7 @@
 		return;
 	}
 
+	preboot_puts("Waiting for PCIe link...\n");
 	preboot_pcie_wait_for_link();
 	preboot_puts("PCIe link is UP!\n");
 }
